optimizely/helpers/validator.py

```diff
@@ -71,7 +80,7 @@ def is_config_manager_valid(config_manager):
     return _has_method(config_manager, 'get_config')
 
 
-def is_event_processor_valid(event_processor):
+def is_event_processor_valid(event_processor: Any) -> bool:
```

Comment:
Please specify type.

Reply:
Let's discuss this offline. I know why you mentioned Any.

Reply:
added types to validator functions with known expected types